x-pack/filebeat/input/httpjson/internal/v2/config_request.go

```diff
@@ -97,6 +98,13 @@ func (c requestConfig) getTimeout() time.Duration {
 	return *c.Timeout
 }
 
+func (c requestConfig) getProxyUrl() string {
```

Comment:
I think we do not need this function, just checking the value of the field should be enough I think.